index.html

```diff
@@ -0,0 +1,313 @@
+        <li class="social-media-icon"><a href="https://www.linkedin.com/in/belay-birhanu-144ba714b/" aria-label="Linkedin"
+            target="_blank" rel="noopener"><i class="fa-brands fa-linkedin-in"></i></a></li>
+        <li class="social-media-icon"><a href="https://github.com/belaymit?tab=repositories" aria-label="Github"
+            target="_blank" rel="noopener"><i class="fa fa-github"></i></a></li>
+        <li class="social-media-icon"><a href="#" aria-label="Angellist" target="_blank" rel="noopener"><i
+              class="fa-brands fa-angellist"></i></a>
+        </li>
+      </ul>
+      <button class="btn">Get My Resume</button>
+    </div>
+    </div>
+    <div class="languages-learned">
+      <ul class="languages">
+         <li class="language-item">
+            <h1>Languages</h1>
+              <a href="#"><i class="fa-solid fa-angle-down"></i></a>
+         </li>
+         <div class="language-body">
+            <div class="language">
+              <div class="language-icon">
+                <img src="./Images/Ellipse 1.png" alt="">
+              </div>
+              <span>JavaScript</span>
+            </div>
+            <div class="language">
+              <div class="language-icon">
+                <img src="./Images/Ellipse_2.png" alt="">
+              </div>
+              <span>HTML</span>
+            </div>
+            <div class="language">
+              <div class="language-icon">
+                <img src="./Images/Ellipse_3.png" alt="">
+              </div>
+              <span>CSS</span>
+            </div>
+         </div>
+         <li class="language-item framework-skills">
+            <h1>Frameworks</h1>
+            <a href=""><i class="fa-solid fa-angle-right"></i></a>
+         </li>
+         <li class="language-item framework-skills">
+            <h1>Skills</h1>
+            <a href=""><i class="fa-solid fa-angle-right"></i></a>
+         </li>
+      </ul>
+    </div>
+  </section>
+  <!-- END OF WORK SECTION  -->
+  <!-- FOOTER SECTION STARTS HERE -->
+  <footer class="footer-section">
+    <div class="contact-header">
+      <h2 class="contact-title">Contact Me</h2>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that
+        needs
+        coding. I’d love to help with it</p>
+    </div>
+    <form class="contact-form" action="https://formspree.io/f/xqkjglvp" method="post">
+      <div class="input-field">
+        <input type="text" class="input" name="name" value="Belay" placeholder="Something..." maxlength="30" required>
+      </div>
+      <div class="input-field">
+        <input type="email" name="email" class="input" value="2belamit@gmail.com" placeholder="someone@example.com"
+          required>
+      </div>
+      <div class="input-field">
+        <textarea class="textarea" name="message" placeholder="Write something here..." maxlength="500"
+          required>This is my portfolio</textarea>
```

Comment:
**OPTIONAL** - For a better user experience, it is not advisable that you have hardcoded input values, using a placeholder as you have already done is good enough. I suggest you remove all hardcoded input values 👍 👍